tfe/resource_tfe_team.go

```diff
@@ -196,6 +205,12 @@ func resourceTFETeamUpdate(d *schema.ResourceData, meta interface{}) error {
 		options.Visibility = tfe.String(v.(string))
 	}
 
+	if v, ok := d.GetOk("sso_team_id"); ok {
+		options.SSOTeamID = tfe.String(v.(string))
+	} else {
+		options.SSOTeamID = tfe.String("")
```

Comment:
This is the else case you mentioned that can be removed once [this](https://github.com/hashicorp/go-tfe/pull/369/files) PR is through right?

Reply:
yah, that is right